docs/content/intro/install.md

```diff
@@ -6,24 +6,128 @@ aliases:
-4. Create a Brigade project: `brig project create`
+* [Install the Brigade CLI](#install-the-brigade-cli)
+* [Install the Brigade Server](#install-the-brigade-server)
+* [Install Brigade Gateways](#install-brigade-gateways)
 
-At this point, you have a running Brigade service. You can use `helm get brigade-server` and other Helm tools to examine your running Brigade server.
+## Prerequisites
 
-## Cluster Ingress
+* A [Kubernetes cluster].  
+  Your cluster should be accessible to the source of your event triggers.
+  For example, if you want to trigger events from GitHub, the cluster should have a public ip address, and a domain name that resolves to the cluster.
+  If you are using Brigade in a local development environment, the [QuickStart] demonstrates how to access a local KinD or Minikube cluster.
+* [Helm] CLI v3+ installed.
+* [kubectl] CLI installed.
+* Free disk space.  
+  The installation requires sufficient free disk space and will fail if your disk is nearly full.
 
-By default, Brigade is not configured with a load balancer service for incoming requests.  Rather, cluster ingress
-comes in the form of one or more [Gateways](../topics/gateways.md) that provide configurable services, usually in tandem
-with ingress resources.
+[Kubernetes cluster]: https://kubernetes.io/docs/setup/
+[Helm]: https://helm.sh/docs/intro/install/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
 
+## Install the Brigade CLI
+
+Install the Brigade CLI, brig, by copying the appropriate binary from our releases page into a directory on your machine that is included in your PATH environment variable.
+
+**linux**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-linux-amd64
+chmod +x /usr/local/bin/brig
+```
+
+**macos**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-darwin-amd64
+chmod +x /usr/local/bin/brig
+```
+
+**windows**
+```powershell
+mkdir -force $env:USERPROFILE\bin
+(New-Object Net.WebClient).DownloadFile("https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-windows-amd64.exe", "$ENV:USERPROFILE\bin\brig.exe")
+$env:PATH+=";$env:USERPROFILE\bin"
+```
+
+The script above downloads brig.exe and adds it to your PATH for the current session.
+Add the following line to your [PowerShell Profile](https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/) to make the change permanent.
+
+```powershell
+$env:PATH+=";$env:USERPROFILE\bin"
+```
+
+## Install the Brigade Server
+
+1. Enable Helm's experimental OCI support by setting the HELM_EXPERIMENTAL_OCI environment variable to 1.
+
+    **bash**
+    ```bash
+    export HELM_EXPERIMENTAL_OCI=1
+    ```
+
+    **powershell**
+    ```powershell
+    $env:HELM_EXPERIMENTAL_OCI=1
+    ```
+
+1. Create a directory to store the Brigade Helm charts.
+
+    **bash**
```

Comment:
Applicable in several places throughout...

Mac users are now likely to be using zsh instead of bash.

I don't have a _strong_ opinion, but I think it's worth at least asking whether we should generalize and change "bash" to "POSIX".